examples/libs/cardano-mesh/basic-tx/main.ts

```diff
@@ -0,0 +1,120 @@
+import { DfnsWallet } from '@dfns/lib-cardano-mesh'
+import { DfnsApiClient } from '@dfns/sdk'
+import { AsymmetricKeySigner } from '@dfns/sdk-keysigner'
+
+import * as CSL from '@emurgo/cardano-serialization-lib-nodejs'
+import {
+  BlockfrostProvider,
+  IFetcher,
+  IInitiator,
+  ISubmitter,
+  Transaction,
+  UTxO,
+} from '@meshsdk/core'
+
+
+import * as dotenv from 'dotenv'
+
+dotenv.config()
+
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+    credId: process.env.DFNS_CRED_ID!,
+    appOrigin: process.env.DFNS_APP_ORIGIN!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  return DfnsWallet.init({
+    walletId: walletId,
+    dfnsClient,
+  })
+}
+
+const hexToBuffer = (hex: string): Buffer => {
+  return Buffer.from(hex.replace(/^0x/, ''), 'hex')
+}
+
+async function main() {
```

Comment:
nit: for slightly better readability, maybe move the `main` function to below all helpers right above when it's called.